src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -244,19 +229,9 @@ public function getExpiresTime()
      *
      * @param int|string|\DateTimeInterface $expire
      *
-     * @throws \InvalidArgumentException
```

Comment:
This is not a "normal" exception: ppl shouldn't catch them, they should bubble and the code should be fixed instead to never hit the situation.
Adding `@throws` usually means "put a try/catch around this calls". This is not the case here, thus the removal.